src/main/java/me/tongfei/progressbar/ProgressBar.java

```diff
@@ -83,11 +84,14 @@ public ProgressBar(
             String unitName,
             long unitSize,
             boolean showSpeed,
-            DecimalFormat speedFormat
+            DecimalFormat speedFormat,
+            ChronoUnit speedUnit,
+            long startFrom,
+            long elapsedSecond
```

Comment:
Argument `long elapsedSecond` should be changed to `Duration elapsed` to better reflect the intent.